lib/state.js

```diff
@@ -90,15 +90,15 @@ export class State extends Map {
     return schemas;
   }
 
-  async build() {
+  async build(config) {
     const buildPlugins = [];
     if (this.has("extensions")) {
-      for (const buildPlugin of this.get("extensions").build) {
-        buildPlugins.push(...(await buildPlugin({ cwd: this.cwd, development: this.development })));
+      for (const buildFunction of this.get("extensions").build) {
+        buildPlugins.push(...(await buildFunction({ cwd: this.cwd, development: this.development, config })))
```

Comment:
I'm a bit tempted to support the non array case here as well since I've yet to see an extension that actually needs to return an array of plugins. Thoughts @leftieFriele? Would be just a simple if not an array, wrap in array type check.

Reply:
Agreed, I'll add that to this use case 👍  it would make it easier to work with, as wrapping things in arrays "just because" often feels a bit weird.

Reply:
I added support for one or many ✅